src/schema/geneInteraction.yaml

```diff
@@ -0,0 +1,467 @@
+id: https://github.com/alliance-genome/agr_persistent_schema/geneInteraction.yaml
+name: Alliance-Gene-Interaction-Schema-Prototype
+title: Alliance Gene Interaction Schema Prototype
+description: Alliance Gene Interaction Schema Prototype with LinkML
+
+default_prefix: alliance
+default_range: string
+
+#default_curi_maps:
+#  - obo_context
+#  - idot_context
+#  - semweb_context
+#  - monarch_context
+
+emit_prefixes:
+  - rdf
+  - rdfs
+  - xsd
+  - skos
+  - owl
+
+imports:
+  - linkml:types
+  - core
+  - reference
+
+prefixes:
+  alliance: 'http://alliancegenome.org'
+  BFO: 'http://purl.obolibrary.org/obo/BFO_'
+  biolink: 'https://w3id.org/biolink/vocab/'
+  CTD: 'http://translator.ncats.nih.gov/CTD_'
+  faldo: 'http://biohackathon.org/resource/faldo#'
+  gff: 'https://w3id.org/gff'
+  linkml: 'https://w3id.org/linkml/'
+  mi: 'http://purl.obolibrary.org/obo/MI_'
+  NLMID: 'https://www.ncbi.nlm.nih.gov/nlmcatalog/?term='
+  OBAN: 'http://purl.org/oban/'
+  RO: 'http://purl.obolibrary.org/obo/RO_'
+  RTXKG1: 'http://kg1endpoint.rtx.ai/'
+  schema: 'http://schema.org/'
+  SEMMEDDB: 'https://skr3.nlm.nih.gov/SemMedDB'
+  SIO: 'http://semanticscience.org/resource/SIO_'
+  # The UMLS Semantic types and groups namespaces don't have a directly
+  # resolvable Base URI, but we point to a pair of sensible web documents
+  UMLSSC: 'https://metamap.nlm.nih.gov/Docs/SemanticTypes_2018AB.txt/code#'
+  UMLSSG: 'https://metamap.nlm.nih.gov/Docs/SemGroups_2018.txt/group#'
+  UMLSST: 'https://metamap.nlm.nih.gov/Docs/SemanticTypes_2018AB.txt/type#'
+  WIKIDATA: 'https://www.wikidata.org/wiki/'            # Wikidata Entity
+  WIKIDATA_PROPERTY: 'https://www.wikidata.org/wiki/Property:'  # Wikidata Property - not a conventional namespace prefix
+
+## ----------
+## CLASSES
+## ----------
+
+classes:
+
+  ## ------
+  ## THINGS
+  ## ------
+
+  Entity:                         # this should be generalized in core.yaml
```

Comment:
I'm not sure we need entity yet - was this copied from biolink-model?  Can we be more restrictive to start with and give interactions gene or protein classes as domain/range for the first phase?

Reply:
Yes, it was copied from biolink. Sure I can remove it.